canvas_sdk/value_set/custom.py

```diff
@@ -0,0 +1,302 @@
+        "444249",
+        "444249",
+        "444944",
+        "444944",
+        "449494",
+        "449496",
+        "451558",
+        "451559",
+        "451560",
+        "453457",
+        "453462",
+        "454178",
+        "454180",
+        "454181",
+        "454205",
+        "454206",
+        "454207",
+        "454371",
+        "545231",
+        "545231",
+        "545232",
+        "545233",
+        "545238",
+        "545239",
+        "545239",
+        "558741",
+        "558745",
+        "559416",
+        "560050",
+        "563304",
+        "563305",
+        "563306",
+        "563310",
+        "564459",
+        "564460",
+        "565068",
+        "565069",
+        "573523",
+        "583982",
+        "583982",
+        "583985",
+        "583985",
+        "590326",
+        "590375",
+        "590376",
+        "591479",
+        "592349",
+        "592421",
+        "594710",
+        "594714",
+    }
+
+
+class LabReportCreatinine(ValueSet):
+    """LabReportCreatinine."""
+
+    VALUE_SET_NAME = "Lab Report Creatinine"
+    EXPANSION_VERSION = "CanvasHCC Update 2018-10-04"
+
+    LOINC = {
+        "2160-0",
+    }
+
+
+class HypertensiveChronicKidneyDisease(ValueSet):
+    """
+    **Clinical Focus:** This value set contains concepts that represent hypertensive kidney disease.
+
+    **Data Element Scope:** This value set may use the Quality Data Model (QDM) category related to Diagnosis.
+
+    **Inclusion Criteria:** Includes only relevant concepts associated with hypertensive kidney disease.
+
+    **Exclusion Criteria:** No exclusions.
+    """
+
+    OID = "2.16.840.1.113883.3.464.1003.109.12.1017"
+    VALUE_SET_NAME = "Hypertensive Chronic Kidney Disease"
+    EXPANSION_VERSION = "eCQM Update 2020-05-07"
+
+    ICD10CM = {"I120", "I129", "I130", "I1310", "I1311", "I132", "I150", "I151"}
+
+    ICD9CM = {
+        "40301",
+        "40310",
+        "40311",
+        "40390",
+        "40391",
+        "40400",
+        "40401",
+        "40402",
+        "40403",
+        "40410",
+        "40411",
+        "40412",
+        "40413",
+        "40490",
+        "40491",
+        "40492",
+        "40493",
+    }
+
+    SNOMEDCT = {
+        "111438007",
+        "123799005",
+        "123800009",
+        "14973001",
+        "193003",
+        "194774006",
+        "194783001",
+        "19769006",
+        "23130000",
+        "28119000",
+        "285831000119108",
+        "285841000119104",
+        "286371000119107",
+        "32916005",
+        "38481006",
+        "39018007",
+        "397748008",
+        "427889009",
+        "428575007",
+        "473392002",
+        "49220004",
+        "57684003",
+        "62240004",
+        "65443008",
+        "66052004",
+        "66610008",
+        "73410007",
+        "78544004",
+        "81363003",
+        "86234004",
+        "90493000",
+    }
+
+
+class KidneyFailure(v2022KidneyFailure):
+    """Added N1830-N1832 per MRHS request."""
```

Comment:
Do we need this customer-customized one?

Reply:
I don't know the history behind it aside from the comments and git history, so I'm just doing a straight port of the existing code. The existing CKD protocol does use this customer-customized one.

Reply:
Let's use the non-customer-customized one starting now.